### ISSUE TITLE
automate release tagging and building

### ISSUE THREAD

**Issue by erikdw** [mesos/storm#83]
We will soon have an actual versioning scheme implemented in this project (see #18, #76, #77).
When #77 is done, we can follow the process documented in #18 to perform releases.

However, it would be way snazzier if we could leverage travis-ci.org to:
1. automatically build the framework artifacts/packages/tarballs
2. tag the code with the version
3. upload the resultant framework packages/tarballs into the github repo

**Comment by salimane** [comment]
@erikdw  i could take of this.

**Comment by erikdw** [comment]
@salimane : that would be great -- hopefully you've followed the discussions in #109 since it is related.

**Comment by salimane** [comment]
@erikdw yep  and I think it's better to start somewhere. The first goal is to magically have these on the releases page:

```
storm-mesos-0.1.0-storm0.10.0-mesos0.28.0.tgz
storm-mesos-0.1.0-storm0.10.0-mesos0.27.1.tgz
storm-mesos-0.1.0-storm0.9.6-mesos0.28.0.tgz
storm-mesos-0.1.0-storm0.9.6-mesos0.27.1.tgz
```

Then, on docker hub, magically have these docker images appear : 
```
mesos/storm-mesos:0.1.0-0.10.0-0.28.0
mesos/storm-mesos:0.1.0-0.10.0-0.27.1
mesos/storm-mesos:0.1.0-0.9.6-0.28.0
mesos/storm-mesos:0.1.0-0.9.6-0.27.1
```

Thanks

**Comment by erikdw** [comment]
@salimane : sounds great.  FYI, you *should* be able to use your own fork of this project for testing the auto-tagging-and-releasing.  TravisCI provides free service to OSS repos.  Not sure about DockerHub.

**Comment by salimane** [comment]
@erikdw yep true

**Comment by erikdw** [comment]
Interestingly, when I merged #117, it automatically closed this issue #83.  That is because the commit message contains the words: `fixes #83`

See this for more info:
 * https://help.github.com/articles/closing-issues-via-commit-messages/
